code/modules/surgery/surgery_step.dm

```diff
@@ -176,6 +177,17 @@
 			chems += chemname
 	return english_list(chems, and_text = require_all_chems ? " and " : " or ")
 
+/datum/surgery_step/proc/get_obj_list()
+	if (!LAZYLEN(obj_needed))
+		return
+	var/list/objs = list()
+	for(var/R in obj_needed)
+		var/datum/obj/temp = GLOB.objs_needed_surgery[R]
+		if(temp)
+			var/objname = temp.name
+			objs += objname
+	return
```

Comment:
You return nothing.
Don't you have to return something? Like, the list..?

Reply:
oh...right that is why probably it was getting a lot of errors, thanks